### PR TITLE
fixes bug (ZEN-TOTAL-DURATION)

### PR DESCRIPTION
**Pull request by KorsaR-ZN**
Fixed a bug that the player does not parse tag ZEN-TOTAL-DURATION

### CODE REVIEW

**Comment by dmlap** [comment]
Thanks! It looks like this change would cause some of our existing test cases to need an update. In [test/manifest](https://github.com/videojs/videojs-contrib-hls/tree/master/test/manifest) there are a bunch of example m3u8 files and a corresponding parsed version that ends in .js instead of .m3u8. Would you be up for giving that a shot?

**Comment by KorsaR-ZN** [comment]
@dmlap Yes. tomorrow I will fix

File: src/m3u8/m3u8-parser.js [hunk 1]
```diff
@@ -126,7 +126,7 @@
     }
 
     // Comments
-    if (line.indexOf('#EXT') !== 0) {
+    if (line.indexOf('#EXT') !== 0 && line.indexOf('#ZEN-') !== 0) {
```
Comment (ntadej) [inline src/m3u8/m3u8-parser.js]: Shouldn't there be `||` instead of `&&`? Your fix changes existing behaviour.

Comment (KorsaR-ZN) [inline src/m3u8/m3u8-parser.js]: @ntadej Not
No. It's all right

example:
```
var line = "#EXT";
console.log((line.indexOf("#EXT") !== 0 && line.indexOf("#ZEN-") !== 0)) // false
console.log((line.indexOf("#EXT") !== 0 || line.indexOf("#ZEN-") !== 0)) // true
```

**Comment by KorsaR-ZN** [comment]
@dmlap Please help me fix tests

**Comment by forbesjo** [comment]
We won't be making anymore changes in master at this time unless they're critical. Could you please open a new PR against development?

**Pull request closed without merging**